composer.json

```
@@ -17,19 +17,19 @@
     ],
     "require": {
         "php": ">=5.6.4",
-        "illuminate/cache": "^5.4 || ^6.0 || ^7.0 || ^8.0 || ^9.0 || ^10.0",
-        "illuminate/contracts": "^5.4 || ^6.0 || ^7.0 || ^8.0 || ^9.0 || ^10.0",
-        "illuminate/queue": "^5.4 || ^6.0 || ^7.0 || ^8.0 || ^9.0 || ^10.0",
-        "illuminate/redis": "^5.4 || ^6.0 || ^7.0 || ^8.0 || ^9.0 || ^10.0",
-        "illuminate/session": "^5.4 || ^6.0 || ^7.0 || ^8.0 || ^9.0 || ^10.0",
-        "illuminate/support": "^5.4 || ^6.0 || ^7.0 || ^8.0 || ^9.0 || ^10.0",
+        "illuminate/cache": "^5.4 || ^6.0 || ^7.0 || ^8.0 || ^9.0 || ^10.0 || ^11.0 || ^12.0",
```

Bei der langen Liste mein ich ja, so langsam könnte man alles unterhalb von 8.0 vermutlich auch mal derzent ignorieren... so mal abgesehen davon, daß ich ohnehin nicht glauben würde, daß von dem älteren Kram dann noch viel funktioniert ;-)
Wenn wir das doch unbedingt alles zu brauchen glauben, ließe sich auch das kürzer fassen:
```suggestion
        "illuminate/cache": ">=5.4 <12.0 || ^12.0",
```